test/Dialect/Torch/canonicalize.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: torch-mlir-opt %s -canonicalize | FileCheck %s
+// RUN: torch-mlir-opt %s -canonicalize --split-input-file | FileCheck %s
```

Comment:
Oof.. this file is 2000+ lines and wasn't using `--split-input-file`? Can you add the `// -----` blocks to the rest of the file? Maybe in a separate PR to keep history cleaner.

Reply:
Follow up would make more sense. We have been incrementally inserting them as models grow. Alternatively someone could write a script to quickly add.